src/runtime/contrib/random/mt_random_engine.cc

```diff
@@ -220,8 +220,7 @@ class RandomEngine {
     }
     if (dtype.bits == 1 || dtype.bits == 4 || dtype.bits == 8 || dtype.bits == 16 ||
         dtype.bits == 32 || dtype.bits == 64) {
-      int num_threads = task.num_threads = runtime::threading::MaxConcurrency();
-      int res = TVMBackendParallelLaunch(ParallelTask::RunTask, &task, num_threads);
+      int res = TVMBackendParallelLaunch(ParallelTask::RunTask, &task, 0);
```

Comment:
It looks like variable `num_threads` from `ParallelTask` structure doesn't use anymore. Should we remove it from the structure?